aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -115,11 +120,17 @@ module aptos_framework::reconfiguration {
             return
         };
 
+        assert!(current_time > config_ref.last_reconfiguration_time, error::invalid_state(EINVALID_BLOCK_TIME));
```

Comment:
Why do we need this check here? Isn't this already done in block::block_prologue?